packages/server/src/http/routes.ts

```diff
@@ -1,5 +1,5 @@
 import Router from '@koa/router'
-import {change1h, change24h, top100} from '../shared.js'
```

Comment:
do not forget to delete the file too

Reply:
![image](https://github.com/user-attachments/assets/5808aaef-30f4-4cbc-b963-db21e716305d)
clearly deleted 🤔